mlflow/gateway/gateway_app.py

```diff
@@ -0,0 +1,78 @@
+import argparse
+from fastapi import FastAPI, Request, HTTPException
+import os
+from traceback import format_exc
+from typing import List
+import uvicorn
+
+from mlflow.version import VERSION
+from mlflow.gateway.constants import CONF_PATH_ENV_VAR
+from mlflow.gateway.handlers import Route, RouteConfig, _route_config_to_route, _load_route_config
+
+
+# Configured and initialized Gateway Routes state index
+ACTIVE_ROUTES: List[Route] = []
+
+app = FastAPI(
+    title="MLflow Model Gateway API",
+    description="The core gateway API for reverse proxy interface using remote inference "
+    "endpoints within MLflow",
+    version=VERSION,
+)
+
+
+@app.get("/health")
+async def health():
+    return {"status": "OK"}
+
+
+@app.get("/gateway/routes/{route_name}")
+async def get_route(route_name: str):
+    filtered = [x for x in ACTIVE_ROUTES if x.name == route_name]
+    if not filtered:
+        raise HTTPException(
+            status_code=404,
+            detail=f"The route '{route_name}' is not present or active on the server. Please "
+            "verify the route name.",
+        )
+    return {"route": filtered[0]}
```

Comment:
```suggestion
    filtered = next(x for x in ACTIVE_ROUTES if x.name == route_name, None)
    if not filtered:
        raise HTTPException(
            status_code=404,
            detail=f"The route '{route_name}' is not present or active on the server. Please "
            "verify the route name.",
        )
    return {"route": filtered}
```

We can just return the first match.

Reply:
good idea